### PR TITLE
add test code (if) at render() in SeventhGame.java

### PR DESCRIPTION
**Pull request by engyengKim**
I add test code (if) to avoid the wrong case that a value is divided by zero.
Check out render() in SeventhGame.java

Please let me know your opinion.
I am looking forward to your feedback or response :)

### CODE REVIEW

**Comment by tonysparks** [comment]
In general, for each different code change you'll want to create a separate branch and pull request for each.  All of these pull requests have the same changes, so if I wanted to approve one of the changes, I'm forced to accept them all at once.

File: src/seventh/client/SeventhGame.java [hunk 1]
```diff
@@ -461,7 +461,10 @@ public void render() {
             gameClock += DELTA_TIME;
         }
         
-        float alpha = (float)(accumulator / step);
+        // avoid wrong case: divided by zero
+        if(step!=0)
```
Comment (tonysparks) [inline src/seventh/client/SeventhGame.java]: This won't compile because `alpha` is enclosed in the `if` scope and therefore is out of scope for use in `renderScreen` function.
